packages/documentation/components/button/button.doc.md

```diff
@@ -4,13 +4,19 @@ It supports multiple variants, sizes, and states.
 
 ## Overview
 
+<ButtonDoc />
```

Comment:
Why are you including same example(ButtonDoc) twice?
here and at line 34?

Reply:
It's for designers. We discussed with Ed that we should put button variations on top for designers to see them right away

Reply:
I think we can leave only one place, the topmost one